developer-docs-site/docs/tutorials/first-transaction-sdk.md

```diff
@@ -304,14 +304,14 @@ Like the previous step, this is another helper step that constructs a transactio
 :!: static/sdks/typescript/examples/typescript/transfer_coin.ts section_5
 ```
 
-Behind the scenes, the `transfer` function generates a transaction payload and has the client sign, send, and wait for it:
+Behind the scenes, the `transfer` function generates a transaction payload and has the client generate, sign, and send a transaction containing it:
 ```ts
 :!: static/sdks/typescript/src/coin_client.ts transfer
 ```
 
-Within the client, <code>generateSignSendWaitForTransaction</code> is doing this:
+Within the client, `generateSignSubmitTransaction` is doing this:
```

Comment:
Does this change work on docusarus?

Reply:
Yep, I checked all the docusaurus stuff.